src/main/java/com/datastax/cdm/feature/WritetimeTTL.java

```diff
@@ -167,11 +173,16 @@ public static List<String> getWritetimeNames(IPropertyHelper propertyHelper) {
         return CqlTable.unFormatNames(propertyHelper.getStringList(KnownProperties.ORIGIN_WRITETIME_NAMES));
     }
 
-    public static Long getCustomWritetime(IPropertyHelper propertyHelper) {
+    protected static Long getCustomWritetime(IPropertyHelper propertyHelper) {
         Long cwt = propertyHelper.getLong(KnownProperties.TRANSFORM_CUSTOM_WRITETIME);
         return null==cwt ? 0L : cwt;
     }
 
+    protected static Long getCustomTTL(IPropertyHelper propertyHelper) {
+		Long cttl = propertyHelper.getLong(KnownProperties.TRANSFORM_CUSTOM_TTL);
+		return null == cttl ? 0L : cttl;
+	}
```

Comment:
nitpick: formatting (indentation) looks incorrect here. could we get that fixed please?